### PR TITLE
SW-4495 Add projects dropdown to accession creation flow, auto apply if there's only 1 project

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1927** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1927?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/1927?utm_source=stack-comment).

File: src/components/accession2/create/Accession2Create.tsx [hunk 1]
```diff
@@ -65,6 +67,23 @@ export default function CreateAccession(): JSX.Element {
 
   const [record, setRecord, onChange] = useForm<AccessionPostRequestBody>(defaultAccession());
 
+  const { availableProjects } = useProjects(record);
+
+  // If there's only 1 project, auto apply it
+  useEffect(() => {
+    if (!availableProjects || availableProjects.length > 1) {
```
[the remark this answers is not in the document]
Comment (nickgraz) [reply src/components/accession2/create/Accession2Create.tsx:74]: Good catch!

File: src/components/accession2/create/Accession2Create.tsx [hunk 2]
```diff
@@ -65,6 +67,23 @@ export default function CreateAccession(): JSX.Element {
 
   const [record, setRecord, onChange] = useForm<AccessionPostRequestBody>(defaultAccession());
 
+  const { availableProjects } = useProjects(record);
```
Comment (karthikbtf) [inline src/components/accession2/create/Accession2Create.tsx:70]: just `useProjects()` since we aren't leveraging anything from the record?

**Review by karthikbtf — APPROVED** [review]
lgtm

File: src/components/accession2/create/Accession2Create.tsx (hunk 2 above)
Comment (nickgraz) [reply src/components/accession2/create/Accession2Create.tsx:70]: Good call

**Review by karthikbtf — APPROVED** [review]

**Pull request merged**